app/src/main/java/com/gdg/android/MainActivity.kt

```diff
@@ -97,44 +99,99 @@ fun Greeting(name: String, depart: String, modifier: Modifier = Modifier) {
         )
 
         Text(
-            text = "24-2 수강 과목",
+            text = stringResource(R.string._24_2),
             modifier = modifier.padding(top = 10.dp),
             color = Color.Black,
         )
+
+        LazyColumn {
+            items(subjects) { subject ->
+                Text(
+                    modifier = Modifier.padding(vertical = 16.dp, horizontal = 5.dp),
+                    text = subject
+                )
+                HorizontalDivider(
+                    modifier = Modifier.fillMaxWidth(),
+                    thickness = 1.dp,
+                    color = Color.LightGray
+                )
+            }
+        }
     }
 }
 
+@Composable
+fun SignInScreen(navController: NavController) {
+    val (department, setDepartment) = remember {
+        mutableStateOf("")
+    }
+
+    val (name, setName) = remember {
+        mutableStateOf("")
+    }
+
+    val context = LocalContext.current
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.White),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
+    ) {
+        Text("안녕하세요, 여러분", fontSize = 30.sp)
+        Spacer(modifier = Modifier.height(16.dp))
+
+        Text("학부", fontSize = 20.sp)
+        TextField(value = department, onValueChange = setDepartment,
+            placeholder = {
+                Text(
+                    "학부를 입력해주세요",
+                    fontSize = 14.sp,
+                    color = Color.Gray
+                )
+            })
+        Spacer(modifier = Modifier.height(16.dp))
+
+        Text("이름", fontSize = 20.sp)
+        TextField(value = name, onValueChange = setName,
+            placeholder = {
+                Text(
+                    "이름을 입력해주세요",
+                    fontSize = 14.sp,
+                    color = Color.Gray
+                )
+            })
+        Spacer(modifier = Modifier.height(80.dp))
+
+        Button(onClick = {
+            if (name.isNotEmpty() and department.isNotEmpty()) {
+                Toast.makeText(context, "로그인에 성공했습니다", Toast.LENGTH_SHORT).show()
+                navController.navigate("greeting")
+            }
+        }) { Text("로그인") }
```

Comment:
여기두 속성명 먼저 적어주고 스트링 추출할 수 있으면 추출하기!!
```suggestion
        }) { Text(text = "로그인") }
```